lib/hexo/post.ts

```diff
@@ -517,8 +515,12 @@ class Post {
     }).then(() => {
       data.content = cacheObj.escapeCodeBlocks(data.content);
       // Escape all Nunjucks/Swig tags
+      let hasTag = true;
```

Comment:
IMHO:

I prefer to make it clear what kind of tag it is in the variable name like `hasSwigTag`.